runtime/doc/api.txt

```diff
@@ -656,8 +656,9 @@ nvim_call_atomic({calls})                                 *nvim_call_atomic()*
 nvim_chan_send({chan}, {data})                              *nvim_chan_send()*
     Send data to channel `id`. For a job, it writes it to the stdin of the
     process. For the stdio channel |channel-stdio|, it writes to Nvim's
-    stdout. For an internal terminal instance (|nvim_open_term()|) it writes
-    directly to terminal output. See |channel-bytes| for more information.
+    stdout. In the case of an internal terminal instance (|nvim_open_term()|)
```

Comment:
"For => In the case" is a random stylistic change that increases verbosity. Don't want to encourage this.